content/content-strategy/audit-content/plan-your-audit/section3.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: text/textblock
+---
+To decide on the scope and type of content audit, you'll need to consider:
+  * the budget
+  * resources you need for a defined timeframe
+  * the actions you want to take after the audit
+
+### [1]3 types of content audit
```

Comment:
Need to double check if this is the correct semantic markup for the heading.

Reply:
Trisha checked - all good